tests/nevermined/test_provenance.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import uuid
+
+
+def new_did():
+    return uuid.uuid4().hex + uuid.uuid4().hex
+
+
+def test_get_provenance(publisher_instance, ddo_sample):
+    logging.debug("".format())
```

Comment:
I don't think this is doing anything, right?

Reply:
Yes, I think that I got it from other test. I will get rid off